SlicerConnector/Startup.cs

```diff
@@ -215,8 +215,9 @@ private async Task HandleWebsocketConnection(HttpContext context, WebSocket webS
 
                     prusaSlicerBroker.DataReceived += async (sender, args) =>
                     {
-                        //var tmp = Encoding.ASCII.GetBytes("{" + args.Data +"}");
-                        //await webSocket.SendAsync(new ArraySegment<byte>(tmp, 0, args.Data.Length), result.MessageType, result.EndOfMessage, CancellationToken.None);
+                        var slicingProgressMessage = new SlicingProgressMessage(args.Data).ToString();
+                        var slicingProgressMessageBytes = Encoding.ASCII.GetBytes(slicingProgressMessage);
+                        await webSocket.SendAsync(new ArraySegment<byte>(slicingProgressMessageBytes, 0, args.Data.Length), WebSocketMessageType.Text, true, CancellationToken.None);
```

Comment:
use slicingProgressMessage.Length instead of args.Data.Length. args.Data.Length is only the payload

Reply:
Done.